### PR TITLE
Collaboration Service Frontend

### PR DESCRIPTION
**Pull request by dedsecrattle**

### CODE REVIEW

File: backend/collaboration-service/user_verification.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
     raise ValueError('USER_SERVICE_URL environment variable not set')
 
 def authenticate(authorization_header) -> str | None:
-    response = requests.get(f"{USER_SERVICE_URL}/auth/verify-token", headers={'Authorization': authorization_header})
+    response = requests.get(f"{USER_SERVICE_URL}/auth/verify-token", headers={'authorization': authorization_header})
+    print(response)
```
Comment (nknguyenhc) [inline backend/collaboration-service/user_verification.py:10]: Please remove printing statements, and use `logging` statements instead.

File: LICENSE [hunk 2]
```diff
@@ -1,21 +0,0 @@
...
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
```
Comment (nknguyenhc) [inline LICENSE:21]: Why removing the license?

**Review by nknguyenhc — COMMENTED** [review]
Please also fix the lint error as per github actions warnings

File: backend/collaboration-service/user_verification.py (hunk 1 above)
Comment (dedsecrattle) [reply backend/collaboration-service/user_verification.py:10]: Yeah i will push the final commit, still working on cursor update

File: LICENSE (hunk 2 above)
Comment (dedsecrattle) [reply LICENSE:21]: I was removing package.json and other file in root directory, didn't saw i deleted it.

will add it

**Review by yitong241 — APPROVED** [review]
LGTM!

**Pull request merged**